pkg/cmd/roachtest/tests/gossip.go

```diff
@@ -338,7 +339,13 @@ func runGossipRestartNodeOne(ctx context.Context, t test.Test, c cluster.Cluster
 	settings := install.MakeClusterSettings(install.NumRacksOption(c.Spec().NodeCount))
 	settings.Env = append(settings.Env, "COCKROACH_SCAN_MAX_IDLE_TIME=5ms")
 
-	c.Start(ctx, t.L(), option.DefaultStartOpts(), settings)
+	// TODO(darrylwong): remove SetDefaultSQLPort once #117125 is addressed.
```

Comment:
Given the below assumption, is the TODO still relevant? i.e., if I understood correctly, this test fails if the invariant `adminUIPort = SQLPort + 1` doesn't hold, right?

Reply:
Yeah that's true, if we change the port assignment logic then it will fail again.

I changed it to discover the port instead, which seems more correct to me anyway.